### ISSUE TITLE
Context for this fork of Insomnia

### ISSUE THREAD

**Issue by ajtran303** [ajtran303/insomnia#1]
# Context for this fork of Insomnia

This repo is a fork of Insomnia, a nice HTTP client that I really like to use. I want to help make it so that more people can use it.

There is a discussion to [Improve accessibility for screen reader users](https://github.com/Kong/insomnia/issues/1436).

@zersiax listed some "low hanging fruit" and I've volunteered to tackle the work on this. 

- The HTTP VERB selector doesn't currently indicate that it can be expanded. It just lists as a button with the label "GET" which can be anything really. Adding aria-expanded to this and toggling it with JS when it toggles would already fix this.
- Various buttons in, for example, the headers tab, as well as various edit fields currently have no label that screenreaders can immediately discover. If there are labels in the UI, aria-labeledby="your_element"here" would already fix this issue. If not, consider adding labels, or add aria-label="blah" to controls that currently only have an icon.

**Comment by ajtran303** [comment]
## My Role

I'm just a volunteer working on this and I cannot promise to be on this full time. But I will try to be as transparent as possible about my work and progress. Currently, I think my time commitment will be limited to weekends.

**Comment by ajtran303** [comment]
I've noticed there is a new UI update that I'm not as familiar with. There is now a dashboard feature. Relatedly, there is an issue opened 2 days ago to address the keyboard navigation for the dashboard. https://github.com/Kong/insomnia/issues/3183

**Comment by ajtran303** [comment]
> keyboard navigation for the dashboard

There is none. I'm going to try exploring an older release to get around this problem. The Insomnia I remember! https://github.com/Kong/insomnia/issues/3178#issuecomment-797358232 

The release is `2020.5.2`.

**Comment by thgcode** [comment]
It's a long time this repo was created, but I created a [branch](HTTPS://github.com/thgcode/insomnia/tree/fix/accessibility-changes) to try to make some fixes. At this time, I managed to add a region for request and response, label some buttons and make the dropdown options detectable with the arrow keys by the screen reader, however this part is a bit buggy... I did not find your code, but I figured I would ask here so I could merge your code on my branch to test it as well.
